doc/whats_new/v0.20.rst

```diff
@@ -113,6 +113,11 @@ Model evaluation and meta-estimators
 - A scorer based on :func:`metrics.brier_score_loss` is also available.
   :issue:`9521` by :user:`Hanmin Qin <qinhanmin2014>`.
 
+- The default of the ``iid`` parameter of :class:`model_selection.GridSearchCV` and
+  :class:`model_selection.RandomizedSearchCV` will change from ``True`` to ``False``
+  in version 0.22, and the parameter will be removed in version 0.24 altogether.
+  :issue:`9085` by :user:`Laurent Direr <ldirer>` and `Andreas Müller`_.
```

Comment:
Add a note that: This parameter is of greatest practical significance where the sizes of different test sets in cross-validation were very unequal, i.e. in group-based CV strategies.